github/test_branch_protection.py

```diff
@@ -0,0 +1,83 @@
+"""
+Check for branch protection consistancy and compliance
+"""
+import helpers
+import resources
+import pyjq
+import pytest
+
+
+
+@pytest.mark.github_org
+@pytest.mark.xfail(reason="no repos for org")
+@pytest.mark.rationale(
+        """
+Consistency check for metadata. If we're examining an org, there should
+be repos we care about in it. (Warning)
+"""
+)
+@pytest.mark.parametrize(
+        "repo_list",
+        [(resources.repos_of_interest_for_org()),],
+)
+def test_repos(repo_list):
```

Comment:
Our standard as been one test per file with the file name matching the test name.

Reply:
Curious as to reason. I can see one concept per file, but I don't see the value an explosion of files provides.

Reply:
discussed, one test per file